lesson_46/HW_L46/demy-app/src/components/Button/Button.jsx

```diff
@@ -1,7 +1,26 @@
 import "./_button.scss";
+import {Buttonbar} from "../Buttonbar";
 
-export function Button({link='#', theme='', children}) {
+function handlerClick(event, func) {
+  if (func) {
+    event.preventDefault();
+    func();
+  } else {
+    return null;
+  }
+}
+
+export function Button({type='link', link='#', theme='', onClick, disabled=false, children}) {
+  const receiveProps = {
+    className: `button ${theme}`,
+    onClick: (event) => handlerClick(event, onClick)
```

Comment:
а почему просто нельзя передать 
```suggestion
    onClick: onClick
```
и уже в самом onClick изначальном делать event.preventDefault?
тут же как раз event будет приходить